app/views/shipit/stacks/show.html.erb

```diff
@@ -21,7 +21,7 @@
     </ul>
   </section>
 
-  <% cache @stack do %>
+  <% cache [@stack, params[:force]] do %>
```

Comment:
this PR is quite old but I guess it is still relevant. Correct me if I'm wrong @JackTLi but you were facing a similar cache issue, right? 

My suggestion in this case is to combine `stack` and `tasks`, so whenever the stack `updated_at` changes or any of tasks (collection) it is going to expire the cache, right @rafaelfranca ?

Reply:
Shipit propagate the `updated_at` changes. If a `task.updated_at` change, it's `task.stack.updated` will be bumped as well.

Reply:
Thanks Jean, should we close this PR?

Reply:
Well, the `force` params still make sense though.

Reply:
I feel like we tried this in production and it didnt work, but it's been so long I'm not sure 🤔 

Something a bit crazier we could try: disabling the cache entirely for a day and seeing if the perf is ok. Since continuous deployment and the cloud move I wonder if old assumptions about perf + usage patterns still apply.

Reply:
I think we can def remove this particular cache since we still have caches for the individual resources

Reply:
Keep in mind that `stack#show` is automatically pulled by everyone that has a tab open on it whenever something change on the stack.

Which mean you can easily get a big spike of hits on it. So as always measure is everyting, but I do believe that higher level cache do see a very decent hit rate, and is much master than the collection of lower caches.